README.md

```diff
@@ -177,7 +162,7 @@ A schema is a definition of a resource used in our HatchifyJS system. We
 use these shared schemas across our backend and frontend to create
 database tables, generate REST endpoints, and create React components
 and data fetchers. Because these schemas are the backbone of our
-frontend and backend HatchifyJS, we will place them in a `schemas/`
```

Comment:
"HatchifyJS" felt out of place here?

Reply:
oops, good catch 😄 
either:
`Because these schemas are the backbone of our frontend and backend, we will...`
or
`Because these schemas are the backbone of our frontend and backend HatchifyJS apps, we will...`
depends how much we want to hammer "HatchifyJS" into their heads!